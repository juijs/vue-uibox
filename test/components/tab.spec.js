import { mount } from 'vue-test-utils'
import UixTab from '@/components/tab'

describe('tab.js', () => {
    const wrapper = mount(UixTab, {
        propsData: {
            items: [
                { text: "Java" },
                { text: "JavaScript" },
                { text: "CSS" },
                { text: "SCSS" }
            ],
            index: 1
        }
    });

    it('snapshot!!', () => {
        wrapper.vm.$nextTick(() => {
            expect(wrapper.vm.$el).toMatchSnapshot();
        });
    });
})