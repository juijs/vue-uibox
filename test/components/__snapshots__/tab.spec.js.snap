// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tab.js snapshot!! 1`] = `
<ul
  class="tab top"
>
  <li>
    <a
      href="#"
    >
      <!---->
      
            Java
        
    </a>
  </li>
  <li
    class="active"
  >
    <a
      href="#"
    >
      <!---->
      
            JavaScript
        
    </a>
    <div
      class="anchor"
    />
  </li>
  <li>
    <a
      href="#"
    >
      <!---->
      
            CSS
        
    </a>
  </li>
  <li>
    <a
      href="#"
    >
      <!---->
      
            SCSS
        
    </a>
  </li>
</ul>
`;
